ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -125,7 +125,7 @@ tests:
       TEST_SCENARIOS: OTA
     test:
     - chain: openshift-e2e-test-qe-longduration
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-by-oc-mirror
```

Comment:
pj-rehearse testing get passed, I think we can revert this change on this line, later I will try to add some dedicated  ci jobs to cover oc-mirror.

Reply:
@jianlinliu Done , thanks